### DISCUSSION TITLE
RuntimeError: [json.exception.type_error.302] type must be number, but is string是什么 问题，如何解决？

### DISCUSSION THREAD

**Discussion by jingmouren** [alibaba/GraphScope#112] [Q&A] [answered]
样例的papers = interactive.execute("g.V().has('author', 'id', 2).out('writes').where(__.in('writes').has('id', 4307)).count()").one()
能够执行，统计节点数与边数也可以
但执行这句sub_graph = interactive.subgraph("g.V().has('year', inside(2014, 2020)).outE('cites')")
时报错
RuntimeError: [json.exception.type_error.302] type must be number, but is string

**Comment by sighingnow** [comment]
Hi @jingmouren, could you provide us more information about the versions of graphscope and vineyard ? You could know the version string by

```python
>>> import vineyard
>>> print(vineyard.__version__)
>>> import graphscope
>>> print(graphscope.__version__)
```

**Comment by jingmouren** [comment]
graphscope:0.1.1
vineyard:0.1.6

**Comment by sighingnow** [comment, marked as answer]
graphscope-0.1.1 requires vineyard-0.1.4. You could install that version via:

``` bash
pip3 install vineyard==0.1.4
```

After reinstall vineyard the problem should be fixed.

**Comment by sighingnow** [comment]
Hi @jingmouren,

I moved this issue here to help others that may encounter the same issue. Free to comment if the above answer still doesn't work for your case.
